recipe/build.sh

```diff
@@ -1,5 +1,14 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-make -f unix/Makefile generic CC=$CC CPP=$CPP
-mkdir -p $PREFIX/bin
-mv zip $PREFIX/bin
+set -eux
+
+export CFLAGS="${CFLAGS} -DLARGE_FILE_SUPPORT -DZIP64_SUPPORT"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
```

Comment:
is this linker flag really needed?

Reply:
we can try without...

Reply:
seems to work (at least on linux64...)

Reply:
yep all back to :heavy_check_mark: , no new warnings as compared to a few weeks ago